dotcom-rendering/src/components/FrontSection.tsx

```diff
@@ -527,17 +528,19 @@ export const FrontSection = ({
 	collectionBranding,
 	isTagPage = false,
 	hasNavigationButtons = false,
+	isAboveDesktopAd = false,
 }: Props) => {
 	const isToggleable = toggleable && !!sectionId;
+	const showVerticalRule = !hasPageSkin;
+	const isBetaContainer = !!containerLevel;
```

Comment:
To make it clear to the reader that if `containerLevel` is used it means that it's a beta container